sqs/src/main/scala/akka/stream/alpakka/sqs/scaladsl/SqsAckFlow.scala

```diff
@@ -202,4 +208,8 @@ object SqsAckFlow {
             }(sameThreadExecutionContext)
       }
       .mapConcat(identity)
+
+  @InternalApi
```

Comment:
How come this is an `@InternalApi` when the implementations in other connectors are not?

Reply:
Fixed now. (They started off as just `private`...)